src/main/java/net/helpscout/api/utils/ParamsUtils.java

```diff
@@ -66,6 +69,19 @@ public static String setFields(String url, List<String> fields) {
         return params;
     }
 
+    public static Map<String, String> getSearchParamsMap(String query, String sortField, String sortOrder, Integer page) throws ApiException {
+        Map<String, String> params = new HashMap<String, String>();
+        params.put("query", query);
+        if (sortField != null && sortField.trim().length() > 0) {
```

Comment:
@bob983 I understand you concern from the beginning:) But sorry, I mean that I did not catch the profit of getting all entries from the map and eliminate null and empty values. From my point of view, to check before will be faster, than iterate over entries then and perform the same checks (2 checks against 4 checks :))). if you concern about code duplication, it is can be eliminated by moving the check to separate method. Am I still missing something?